pkg/objstore/oci/helper.go

```diff
@@ -0,0 +1,270 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
+// Licensed under the Apache License 2.0.
```

Comment:
This needs to be different: https://github.com/thanos-io/thanos/blob/main/scripts/copyright/copyright.go#L15-L21. When you are adding PRs to Thanos then you become a Thanos author (: You still retain the copyright which in this case is Oracle but just the header needs to be a bit different. I hope that's OK. Also, IANAL.

```suggestion
// Copyright (c) The Thanos Authors.
// Licensed under the Apache License 2.0.
```

Reply:
Update.
Please let me know if there is anything else that need me to adjust. THX!

Reply:
Hi @GiedriusS 
I have rebased my PR and updated the oci-go-sdk. Please let me know if the latest looks good to you.
Thx!